07-javascript.js

```diff
@@ -95,14 +107,20 @@ if (false) {
 //
 
 // START
+function Person(newName, newTitle){
+  this.name = newName,
+  this.title = newTitle,
+  this.fullName = function(){
+    return this.title + " " + this.name;
+  }
```

Comment:
We talked in class about the downsides of storing instance methods directly in the objects, and instead favored adding them to the prototype

~~~ js
Person.prototype.fullName = function ...
~~~

No deduction, because it passes the tests.